R/workflow.R

```diff
@@ -51,6 +51,7 @@ Workflow <- R6::R6Class(
 
       # Check of Workflow inputs
       self$simulationStructures <- list()
+      simulationSets <- c(simulationSets)

```

Comment:
@pchelle I don't want to always have to create a list for one model set. Does it make sense?